product.json

```diff
@@ -36,6 +36,7 @@
 		{
 			"name": "ms-vscode.js-debug-companion",
 			"version": "1.0.18",
+			"sha256": "b49ee134f452c88fe3de09ae62b7f77aa66d1d6dcd794c48e065bdc2c74d4a30",
```

Comment:
@connor4312 @sandy081 fyi this adds a `sha256` attribute to our `builtInExtensions` in `product.json` as a way to validate the bits when downloading.